modules/javafx.graphics/src/test/resources/test/com/sun/javafx/css/checksum.css

```diff
@@ -0,0 +1,140 @@
+
```

Comment:
This file should have a copyright header.

Reply:
Added copyright header.